src/components/StatusBadge/StatusBadge.mdx

```diff
@@ -34,25 +36,41 @@ import Link from "../Link/Link";
 
 ## Accessibility
 
-The visual aspect of the `StatusBadge` component displays three different status
-levels through the use of three different colors. It is important to assign
-the appropriate text so that the user can understand the priority level of the
-`StatusBadge`, and not rely just on the color.
+The visual aspect of the `StatusBadge` component displays multiple different
+statuses and priorities through the use of different colors, so it is
+recommended to align the usage of the component with the semantic meaning of one
+of the component's variants. 
 
-## Patterns
+However, when using the `StatusBadge` component, the implementation should not
+rely on the color alone to convey meaning, as color is not a universal
+experience for everyone. It is important that the meaning of the label aligns
+with the general meaning associated with the semantic colors used in a specific
+variant. In this way, the user will be able to understand the priority of the
+`StatusBadge` and not rely just on the color.
 
-### Higher Barrier
+## Semantic Type Variants
 
-Used in instances where the associated message is the highest barrier. Use
-`"high"` and `"medium"` for these levels.
+The `semantic` variants align with specific uses. Please refer to the purpose of each variant.
 
-<Canvas of={StatusBadgeStories.HigherBarrier} />
+<Canvas of={StatusBadgeStories.TypeVariants} />
 
-### Lower Barrier
+## Level Variants (deprecated)
 
-Used in instances where the associated message is the lower barrier. Use
-`"low"` for this level which is the default.
+The `level` variants have been deprecated in favor of the semantic `type`
+variants. However, for backward compatability, the `level` prop will override
+the `type` prop.
 
-<Canvas of={StatusBadgeStories.LowerBarrier} />
+<Canvas of={StatusBadgeStories.LevelVariants} />
+
+## Labeling Variations
+
+The capitalization of a badge label will be controlled by the component and
```

Comment:
This section is a little confusing. "The capitalization of a badge label will be controlled by the component" implies to me that there is a prop in the component that determines capitalization. Unless I'm misunderstanding, I think all that needs to be here is "Labels will render exactly as they are passed into the component. If a use case calls for all caps, then an all caps string should be passed into the component."

Reply:
Yes, that could be better.  In fact, the word `not` was missing from one spot, which changed the whole meaning.  I just rewrote this description.